src/main/java/embl/ebi/variation/commons/models/converters/data/VariantReadConverter.java

```diff
@@ -0,0 +1,26 @@
+package embl.ebi.variation.commons.models.converters.data;
+
+import com.mongodb.DBObject;
+import org.opencb.biodata.models.variant.Variant;
+import org.opencb.opencga.storage.core.variant.VariantStorageManager;
+import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantConverter;
+import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantSourceEntryConverter;
+import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantStatsConverter;
+import org.springframework.core.convert.converter.Converter;
+
+/**
+ * Created by tom on 21/09/16.
+ */
+public class VariantReadConverter implements Converter<DBObject, Variant> {
```

Comment:
Yes, it's only an example that is not followed a pair of sections below: http://docs.spring.io/spring-data/mongodb/docs/current/reference/html/#mongo.converter-disambiguation